Examples/Java/Sources/Board.java

```diff
@@ -466,7 +480,7 @@ public void mergeFrom(Board model) {
         final private TypeAdapter<Set<User>> set_User_TypeAdapter;
         final private TypeAdapter<String> stringTypeAdapter;
 
-        public BoardTypeAdapter(Gson gson, BoardTypeAdapterFactory factory, TypeToken typeToken) {
+        public BoardTypeAdapter(@NonNull Gson gson, BoardTypeAdapterFactory factory, TypeToken typeToken) {
```

Comment:
@RicoYao i'm unsure about the nullability of the other params.

Reply:
Probably best not to annotate them. I'm not seeing any obvious reliance on them being non-null in Gson's implementation of `getDelegateAdapter(factory, typeToken)`